test/formol/types/datepicker.test.jsx

```diff
@@ -0,0 +1,92 @@
+import { mount } from 'enzyme'
+import React from 'react'
+
+import Formol, { Field } from '../../../src'
+
+const nativeType = 'text' // react-datepicker uses input with type='text'
+const rawValue1 = '1988-05-27'
+const rawValue2 = '2121-12-12'
+const title = 'Date'
+const type = 'date'
+const value1 = '05/27/1988'
+const value2 = '12/12/2121'
+
+describe('Datepicker field', () => {
+  it('handles changes', async () => {
+    const onSubmit = jest.fn()
+    const wrapper = mount(
+      <Formol onSubmit={onSubmit} item={{ [type]: rawValue1 }}>
+        <Field name={type} type={type}>
+          {title}
+        </Field>
+      </Formol>
+    )
+    const input = () =>
+      wrapper
+        .find('Field')
+        .find('input')
+        .first()
+    const submit = () => wrapper.find('.Formol_Formol__submit')
+    const cancel = () => wrapper.find('.Formol_Formol__cancel')
+
+    expect(submit().props().disabled).toBeTruthy()
+    expect(cancel().props().disabled).toBeTruthy()
+    expect(input().props().type).toEqual(nativeType)
+    expect(input().props().value).toEqual(value1)
+
+    await input().simulate('focus')
```

Comment:
This line causes an `UnhandledPromiseRejectionError`. Without it, all tests are passed.